stubs/config.php

```diff
@@ -70,4 +70,12 @@
         //  ],
     ],
 
+    'requirements' => [
```

Comment:
Can you also add a comment block explaining with this is, like we have with the other arrays?